doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md

```diff
@@ -11,7 +11,10 @@ sort_idx: 10
 * TOC
 {:toc}
 
-This guide describes how to use *C SDK* on linux-based systems.
+The application code based on Kaa C SDK is similar for most of the platform used.
+However, build system is not.
+
+Throughout the guide, we focus on details of the CMake-based build system that is used to tie application code together with the C SDK.
```

Comment:
may be the word `connect` is more appropriate usage than `tie` from the technical point of view?